apiserver/facades/client/spaces/spaces.go

```diff
@@ -35,19 +36,22 @@ type ControllerConfigService interface {
 type NetworkService interface {
 	// AddSpace creates and returns a new space.
 	AddSpace(ctx context.Context, space network.SpaceInfo) (network.Id, error)
-	// Space returns a space from state that matches the input ID.
-	// An error is returned if the space does not exist or if there was a problem
-	// accessing its information.
+	// Space returns a space from state that matches the input ID. If the space
+	// is not found, an error is returned satisfying networkerrors.SpaceNotFound.
```

Comment:
Would you mind updating this to `[github.com/juju/juju/domain/network/errors.SpaceNotFound]`